error.go

```diff
@@ -77,11 +80,23 @@ func ErrorAttrs(_ ...errorAttrsOption) func(groups []string, attr slog.Attr) slo
 		}
 		attrs = append(attrs, slog.Any(a.Key, errorNoAttrs{err: err}))
 
-		var key string
-		if len(groups) > 0 {
-			key = a.Key
+		cfg := &errorAttrsConfig{}
+		for _, opt := range opts {
```

Comment:
> It looks that this line `cfg := &errorAttrsConfig{}` will stay here anyway.

Why?

BTW, probably it makes sense to avoid pointer to config struct - to make sure it won't change by `key()` call below.